docs/_template.html

```diff
@@ -2,84 +2,102 @@
-          integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-    <script type="text/javascript" src="{{root}}content/fsdocs-tips.js"></script>
-    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
-    <!--[if lt IE 9]>
-      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
-    <!-- BEGIN SEARCH BOX: this adds support for the search box -->
-    <link rel="stylesheet"
-          href="https://cdnjs.cloudflare.com/ajax/libs/JavaScript-autoComplete/1.0.4/auto-complete.css"/>
-    <!-- END SEARCH BOX: this adds support for the search box -->
-    {{fsdocs-watch-script}}
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="author" content="{{fsdocs-authors}}">
+  <title>{{fsdocs-page-title}}</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Reem+Kufi:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+  <link rel="shortcut icon" type="image/x-icon" href="{{root}}images/favicon.ico">
+  <link type="text/css" rel="stylesheet" href="{{root}}content/fsdocs-custom.css" />
+  <link type="text/css" rel="stylesheet" href="{{root}}content/fsdocs-default.css" />
+  <script type="text/javascript" src="{{root}}content/fsdocs-tips.js"></script>
+  <!-- BEGIN SEARCH BOX: this adds support for the search box -->
+  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
+  <link rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/JavaScript-autoComplete/1.0.4/auto-complete.css" />
+  <!-- END SEARCH BOX: this adds support for the search box -->
+  {{fsdocs-watch-script}}
 </head>
 
 <body>
-<main>
-    <div class="flex-shrink-0 p-3 bg-white" style="width: 280px;">
-        <a href="/" class="d-flex align-items-center pb-3 mb-3 link-dark text-decoration-none border-bottom">
-            <svg class="bi me-2" width="30" height="24">
-                <use xlink:href="#bootstrap"/>
-            </svg>
-            <span class="fs-5 fw-semibold">Fantomas</span>
-        </a>
-        <a href="{{fsdocs-logo-link}}"><img id="fsdocs-logo" src="{{root}}/{{fsdocs-logo-src}}"/></a>
-        <!-- BEGIN SEARCH BOX: this adds support for the search box -->
-        <div id="header">
-            <div class="searchbox" id="fsdocs-searchbox">
-                <label for="search-by">
-                    <i class="fas fa-search"></i>
-                </label>
-                <input data-search-input="" id="search-by" type="search" placeholder="Search..."/>
-                <span data-search-clear="">
-                        <i class="fas fa-times"></i>
-                    </span>
+  <main id="main-container">
```

Comment:
You probably want to flex things a little bit different on mobile.

![image](https://user-images.githubusercontent.com/2621499/180732177-70d1c666-7e70-4380-8005-e7cf1fb22e91.png)

I think there it makes more sense that the menu is over the main content.